src/main/scala/com/fortysevendeg/scala/android/modules/utils/validation.scala

```diff
@@ -0,0 +1,40 @@
+package com.fortysevendeg.scala.android.modules.utils
+
+import android.widget.EditText
+import com.fortysevendeg.scala.android.R
+import macroid.ContextWrapper
+import com.fortysevendeg.macroid.extras.ResourcesExtras._
+
+import scalaz._
+import scalaz.std.list._
+import scalaz.syntax.traverse._
+
+object validation {
+
+  type Validator[T] = (T) => Validation[ValidationMessage, T]
```

Comment:
We should not be using type aliases or wrappers to demonstrate this functionality as it hides the true power behind `ValidationNel`